simapp/go.mod

```diff
@@ -222,6 +225,20 @@ require (
 // replace (
 // 	<temporary replace>
 // )
+// FIXME: remove temporary replace; replaced since published package uses tendermint protos (local copy uses cometbft v1)
+replace cosmossdk.io/store => ../store
```

Comment:
Since this are each separate modules, seems like we could have split this into smaller easier to review PRs.